include/cuco/static_set_ref.cuh

```diff
@@ -71,6 +71,8 @@ class static_set_ref
                       ProbingScheme>,
     "ProbingScheme must inherit from cuco::detail::probing_scheme_base");
 
+  // TODO static_assert that StorageRef::extent_type is valid_extent
```

Comment:
Needs to be done in this PR

Reply:
Addressed in [9f01917](https://github.com/NVIDIA/cuCollections/pull/315/commits/9f0191755b686b3a6a5015a990aacc3ba2a2b006)